geoips/commandline/ancillary_info/alias_mapping.yaml

```diff
@@ -74,12 +74,8 @@ sectors:
   - sector
 package:
   - pkg
-  - pkgs
-  - packages
 packages:
-  - pkg
   - pkgs
-  - package
```

Comment:
Why edit this? I think there was probably a reason for all of these aliases.

Reply:
This was from Evan's PR that just got merged - it looks like the alias just got split up in the list between "package" and "packages" - so singular aliases went with singular and plural went with plural